spec/models/applicant_spec.rb

```diff
@@ -52,8 +53,19 @@
     expect(subject).to be_valid
   end
 
-  it 'is valid  when email address is not provided' do
+  it 'is valid when email address is not provided' do
     subject.email_address = nil
     expect(subject).to be_valid
   end
+
+  it 'is not valid when a national insurance number is not provided' do
+    subject.national_insurance_number = nil
+    expect(subject).to_not be_valid
```

Comment:
Add include clause as to match the next test.
Without it this test may pass if the model is invalid for another reason.

Reply:
☝️ What Charlie said.